app/cdash/tests/test_consistenttestingday.php

```diff
@@ -128,5 +129,23 @@ public function testConsistentTestingDay()
         $buildgroup = array_pop($jsonobj['buildgroups']);
         $this->assertEqual(1, count($buildgroup['builds']));
         $this->assertEqual('testing_day_upload_file', $buildgroup['builds'][0]['buildname']);
+
+        // Change nightly time & time zone for this project again.
+        $this->createProject([
+            'Id' => $this->project->Id,
+            'Name' => 'ConsistentTestingDay',
+            'NightlyTime' => '18:00:00 America/Denver'
+        ], true);
+
+        // Submit build files.
+        $this->submission('ConsistentTestingDay', "$dir/Build_3.xml");
+        $this->submission('ConsistentTestingDay', "$dir/Configure_3.xml");
+        $this->submission('ConsistentTestingDay', "$dir/Upload_3.xml");
+
+        // Verify the expected start and end time for this build.
+        $matching_builds = DB::table('build')->where('name', '=', 'testing_day_3')->get();
```

Comment:
It would be better to use Eloquent for this.  Something like this should work:
```suggestion
        $matching_builds = Build::where('name', '=', 'testing_day_3')->get();
```